### DISCUSSION TITLE
File upload above 1MB

### DISCUSSION THREAD

**Discussion by miggio90210** [feathersjs/feathers#3179] [Q&A]
I'm really struggling with load a file using last version of featherjs. 
I have to compile a form that include a file to be uploaded and if file size is below 1MB there is no problem, otherwise feathers ignore the service I want to invoke and it uses authentication  without any reason.
I tried to set koabody jsonLimit to 100 but it doesn't work

**Comment by gustojs** [comment]
I can confirm that the same limit exists for me.
